src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterEnterOnTokenTests.cs

```diff
@@ -1365,7 +1365,7 @@ void Main(object o)
                 Assert.True(
                     CSharpIndentationService.ShouldUseSmartTokenFormatterInsteadOfIndenter(
                         Formatter.GetDefaultFormattingRules(workspace, root.Language),
-                        root, line.AsTextLine(), document.Options, CancellationToken.None));
+                        root, line.AsTextLine(), await document.GetOptionsAsync(CancellationToken.None), CancellationToken.None));
```

Comment:
presumably GetOptionsAsync takes an optional cancellation token.  If so, feel free to pass nothing in tests.